### PR TITLE
workload for "hands on openshift 4.8" new class

### PR DESCRIPTION
**Pull request by newgoliath**
##### SUMMARY

1. adds hands_on_openshift_48 workload 
  creates a few projects and deploys some sample apps
1. Fixes ocp4_workload_logging version_compare issues
  No longer supports OpenShift <4.6

##### ISSUE TYPE
- New role Pull Request
- Role bug fix pull request

##### COMPONENT NAME
* ocp4_workload_hands_on_openshift_48
* ocp4_workload_logging

### CODE REVIEW

**Review by stencell — APPROVED** [review]
We're gonna need a bigger boat.

**Review by varodrig — APPROVED** [review]

File: ansible/roles_ocp_workloads/ocp4_workload_hands_on_openshift_48/files/json_logs_pod.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    component: test
+  name: json-log-generator
+  namespace: json-logs-example
+code: &code-str |
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_hands_on_openshift_48/files/json_logs_pod.yaml:8]: Let's please avoid yaml anchors. This is weird and scary :fearful:

File: ansible/roles_ocp_workloads/ocp4_workload_hands_on_openshift_48/files/json_logs_pod.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    set_vals()
+    print json.dumps(data)
+    sys.stdout.flush()
+    time.sleep(0.1) # sleep for 100 msec
+spec:
+  containers:
+    - args:
+        - python2
+        - -c
+        - *code-str
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_hands_on_openshift_48/files/json_logs_pod.yaml:32]: Can just put the code block quote here :smile:

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: ansible/roles_ocp_workloads/ocp4_workload_hands_on_openshift_48/tasks/workload.yml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    - pipelines_namespace.yaml
+    - pipelines_role_binding_argocd_edit.yaml
+
+# skip this?
+#- name: Configure ArgoCD
+#  k8s:
+#    state: present
+#    definition: "{{ lookup('file',  item ) | from_yaml }}"
+#  loop:
+#  - argocd_app_of_apps.yaml
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_hands_on_openshift_48/tasks/workload.yml:109]: What's the story with this?

**Pull request merged**